src/docs/self-hosted/custom-ca-roots.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: "Self-Hosted Custom CA Roots"
+---
+
+Starting with Sentry `21.7.0`, if you need to have Sentry access services which do not have TLS certificates from publicly trusted CA roots, it's now possible to easily add them to the containers. Just add the certificates to the `certificates` folder inside the root of your Sentry install and restart the containers.
+
+<Alert title="Note" level="info">
+  While you can run <code>update-ca-certificates</code> in each container, that will update the system's root bundle on disk, but does nothing for any copies in memory. Restarting the container will update the bundle and make sure it's used.
+</Alert>
+
+The container's logs will have the output from `update-ca-certificates` right at the start if there is a problem with a given certificate.
+
+## Dependencies with bundled roots
+
+Some dependencies have opted to bundle their own CA roots and ingore the system CA roots. Where known, they have been configured to use the system roots. If something seems to ignore the system roots, make an issue so it can be tracked down and fixed.
```

Comment:
```suggestion
Some dependencies have opted to bundle their own CA roots and ignore the system CA roots. Where known, they have been configured to use the system roots. If something seems to ignore the system roots, make an issue so it can be tracked down and fixed.
```